Classes/Util.php

```diff
@@ -330,6 +330,7 @@ public static function getConfigurationFromPageId(
                     $initializedPageSelect = true;
                 }
 
+                    /** @var $tmpl \TYPO3\CMS\Core\TypoScript\ExtendedTemplateService */
```

Comment:
Nice hint! I'll install that, but i would maybe think about to change this in the later versions to make the start easier for others, we can also use the ::name Variante then when we drop 6.2.x support. I'll remove it for now.